wsd/DocumentBroker.hpp

```diff
@@ -611,7 +611,7 @@ class DocumentBroker : public std::enable_shared_from_this<DocumentBroker>
     /// Upload the doc to the storage.
     void uploadToStorageInternal(const std::shared_ptr<ClientSession>& session,
                                  const std::string& saveAsPath, const std::string& saveAsFilename,
-                                 const bool isRename, const bool force);
+                                 const bool isRename, const bool isExport, const bool force);
```

Comment:
can be done via enum? (  SaveAs | Rename | Export )

Reply:
It seems we are passing this flag around only to generate the notification message to the client to be "exportas: url=..." instead of "saveas: url=...", but it's not clear to me why we need this distinction at all. Let's discuss please.

Reply:
`save as` closes old document and opens the newly exported file, `export` only exports the file and informs user about the result. Difference is visible in the client side where message handling happens

Reply:
Thank you. This answers my question above. 

Regarding using an enum, yes, this function was already messy and now it's more complicated. I had plans on cleaning it up. Now there is even more reason.